.github/workflows/e2e-webpack-workflow.yml

```diff
@@ -0,0 +1,110 @@
+
+        mkdir src
+        echo "import _ from 'lodash';function printHello() { console.log(_.join(['Hello', 'webpack'], ' '))}; printHello();" | tee src/index.js
+
+        yarn webpack
+        [[ "$(node dist/main.js)" = "Hello webpack" ]]
+
+        rm -rf dist src webpack.config.js
+
+        # raw-loader (imported)
+        yarn add raw-loader
+
+        mkdir src
+        echo 'import text from "raw-loader!./text.txt"; console.log(text);' | tee src/index.js
+        echo 'Hello raw-loader' | tee src/text.txt
+
+        yarn webpack
+        [[ "$(node dist/main.js)" = "Hello raw-loader" ]]
+
+        rm -rf dist src webpack.config.js
+
+        # ts-loader
+        yarn add -D ts-loader typescript pnp-webpack-plugin @types/lodash
+
+        echo "const PnpWebpackPlugin = require('pnp-webpack-plugin'); module.exports = {mode: 'none', entry: './src/index.ts',output: { filename: 'main.js'}, resolve: { extensions: ['.ts', '.tsx', '.js']},module: { rules: [ { test: /\.tsx?$/, loader: require.resolve('ts-loader'),options: PnpWebpackPlugin.tsLoaderOptions({/* ... regular options go there ... */}) }  ]}};" | tee webpack.config.js
+
+        echo "{\"compilerOptions\": {  \"noImplicitAny\": true,  \"removeComments\": true,  \"preserveConstEnums\": true,  \"sourceMap\": true}}" | tee tsconfig.json
+
+        mkdir src
+        echo "import * as _ from 'lodash';function printHello() { console.log(_.join(['Hello', 'ts-loader'], ' '))}; printHello();" | tee src/index.ts
+
+        yarn webpack
+        [[ "$(node dist/main.js)" = "Hello ts-loader" ]]
+
+        rm -rf dist src webpack.config.js tsconfig.json
+
+        # less-loader
+        yarn add -D less less-loader css-loader style-loader file-loader bootstrap-less
+
+        echo "const path = require('path'); module.exports = { mode: 'none', entry: './src/index.js', output: { filename: 'main.js', path: path.resolve(__dirname, 'dist') }, module: { rules: [ { test: /\.less$/, use: ['style-loader', 'css-loader', 'less-loader'] }, { test: /\.(png|svg|jpg|gif)$/, use: ['file-loader'] }, { test: /\.(woff|woff2|eot|ttf|otf)$/, use: ['file-loader'] } ] } }; " | tee webpack.config.js
+
+        mkdir src
+        echo "import './main.less';" | tee src/index.js
+        echo "@import '~bootstrap-less/bootstrap/index.less';@import './other.less';.box:extend(.hotpink) {width: 200px;height: 200px;}" | tee src/main.less
+        echo ".hotpink {background: hotpink;}" | tee src/other.less
+
+        yarn webpack
+
+        ls dist | grep "main.js"
+        ls dist | grep ".svg"
+        ls dist | grep ".woff2"
+
+        rm -rf dist src webpack.config.js
+
+        # thread-loader, babel-loader and sass-loader (source: https://github.com/webpack-contrib/thread-loader/tree/master/example)
+        yarn add -D thread-loader babel-loader  @babel/core babel-preset-env @babel/plugin-proposal-object-rest-spread node-sass sass-loader css-loader style-loader react lodash-es bootstrap-scss
```

Comment:
I think `sass` (instead of `node-sass`) would work without unplug - but it's still nice to test `node-sass`, as it has 7x the number of downloads.